datatypes/src/operations/image/colorizer.rs

```diff
@@ -288,7 +288,17 @@ impl Colorizer {
         let breakpoints = match self {
             Self::LinearGradient { breakpoints, .. }
             | Self::LogarithmicGradient { breakpoints, .. } => breakpoints,
-            _ => unimplemented!("Must never call `color_table` for types without breakpoints"),
+
+            _ => {
+                // return empty color table for potential wrong usage
+
+                debug_assert!(
+                    false,
+                    "Must never call `color_table` for types without breakpoints"
+                );
+
+                return Vec::new();
```

Comment:
why don't you change the return type of this method to `Result`? Would be much cleaner.

Reply:
This isn't called in practice and now it just does not panic.